### DISCUSSION TITLE
Apple Private Key, Maps ID, MapKit JS

### DISCUSSION THREAD

**Discussion by somegirlonmac** [codemagic-ci-cd/codemagic-docs#1612] [Q&A] [answered]
Hello!
I would like to ask if there is a way to get an Apple private key, Maps ID, and MapKit JS without subscribing to a paid Apple developer/enterprise program?

Thank you!

**Comment by icarusdust** [comment, marked as answer]
Hi @somegirlonmac, you will have to subscribe to the Apple Developer account to build ios apps.

**Reply by somegirlonmac** [reply]
Got this. Thank you @icarusdust :D
